From akpm@linux-foundation.org  Fri Oct 31 10:04:31 2008
From: Anton Vorontsov <avorontsov@ru.mvista.com>
Date: Wed, 29 Oct 2008 14:25:50 -0700
Subject: USB: protect hcd.h from multiple inclusions
To: greg@kroah.com
Cc: linux-usb@vger.kernel.org, akpm@linux-foundation.org, avorontsov@ru.mvista.com
Message-ID: <200810292125.m9TLPo0i021694@imap1.linux-foundation.org>


From: Anton Vorontsov <avorontsov@ru.mvista.com>

This will let us use this header in other header files.
Will be needed for the FHCI USB Host driver.

Signed-off-by: Anton Vorontsov <avorontsov@ru.mvista.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/hcd.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/drivers/usb/core/hcd.h
+++ b/drivers/usb/core/hcd.h
@@ -16,6 +16,8 @@
  * Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
+#ifndef __USB_CORE_HCD_H
+#define __USB_CORE_HCD_H
 
 #ifdef __KERNEL__
 
@@ -490,3 +492,5 @@ extern struct rw_semaphore ehci_cf_port_
 extern unsigned long usb_hcds_loaded;
 
 #endif /* __KERNEL__ */
+
+#endif /* __USB_CORE_HCD_H */
